doc/DISTUTILS.rst.txt

```diff
@@ -361,6 +361,9 @@ during the build process are ready to be compiled. This form
 of generic typing is also supported for C header files (preprocessed to
 produce :file:`.h` files).
 
+Additionally, NumPy distutils supports automatic conversion of source files named *.src". This facility can be used to maintain very similar code blocks requiring only simple changes between blocks. During the build phase of setup, if a template file named <somefile>.src is encountered, a new file named <somefile> is constructed from the template and placed in the build directory to be used instead. Two forms of template conversion are supported. The first form occurs for files named <file>.ext.src where ext is a recognized Fortran extension (f, f90, f95, f77, for, ftn, pyf). The second form is used for all other cases. Read more `here`_.
+
+.. _here: http://www.numpy.org/devdocs/reference/distutils.html#conversion-of-src-files
```

Comment:
This is a good start. You should remove lines 303 - 363 from here and merge them with the content you linked too

Reply:
I removed lines 308 to 363, and at the end of the first paragraph I added a link to this page: http://www.numpy.org/devdocs/reference/distutils_guide.html#template-files. Is this what you wanted? I would add the file here but git is not allowing me to for some reason.

Reply:
This is how it looks afterwards

![image](https://user-images.githubusercontent.com/39683881/55225966-dc0d8d80-5239-11e9-8418-487d6616fe2b.png)

Reply:
> I would add the file here but git is not allowing me to for some reason

What file are you adding? You should be modifying the file in-place inside your clone of a git repo.

Reply:
yeah, I did, but I didn't push it yet